specs/~dynamic-names.json

```diff
@@ -1,43 +1,53 @@
+  "overview": "Dynamic Names are a special way to dynamically refer to a tag name.\n\nDynamic Names MUST be a non-whitespace character sequence NOT containing\nthe current closing delimiter.\n\nThis tag's content refers to a key in the context whose value will be used in\nplace of the Dynamic Name itself as content of the tag. The name resolution is\nas follows:\n  1) Split the name on periods; the first part is the name to resolve, any\n  remaining parts should be retained.\n  2) Walk the context stack from top to bottom, finding the first context\n  that is a) a hash containing the name as a key OR b) an object responding\n  to a method with the given name.\n  3) If the context is a hash, the data is the value associated with the\n  name.\n  4) If the context is an object, the data is the value returned by the\n  method with the given name.\n  5) If any name parts were retained in step 1, each should be resolved\n  against a context stack containing only the result from the former\n  resolution.  If any part fails resolution, the result should be considered\n  falsey, and should interpolate as the empty string.\nThe resolved data should be coerced into a string before being used as\ncontent.\n\nSet Delimiter tags MUST NOT affect the resolution of a Dynamic Name. The\nDynamic Names MUST be rendered against the context stack local to the tag.\nFailed resolutions of the key (context lookups) should be considered falsey\nand should interpolate as the empty string.\n\nDynamic Names can be combined with every other tag: those tags' content is the\nresolution of the Dynamic Name. Dynamic Names cannot be resolved more than\nonce (Dynamic Names cannot be nested).\n",
   "tests": [
     {
-      "name": "Basic Behavior",
+      "name": "Basic Behavior - Interpolation",
+      "desc": "The asterisk operator is used for dynamic names.",
+      "data": {
+        "dynamic": "Hello, world!"
+      },
+      "template": "\"{{*dynamic}}\"",
+      "partials": {},
+      "expected": "\"Hello, world!\""
```

Comment:
with data in current test, {{*dynamic}} should be processed as {{Hello, world}} so expected should be ""
or have data: { dynamic: greeting, greeting: "Hello, world" }